creator-node/src/config.js

```diff
@@ -283,21 +283,25 @@ const config = convict({
     default: true
   },
 
-  // Transcoding settings
+  /** Upload settings */
   transcodingMaxConcurrency: {
     doc: 'Maximum ffmpeg processes to spawn concurrently. If unset (-1), set to # of CPU cores available',
     format: Number,
     env: 'transcodingMaxConcurrency',
     default: -1
   },
-
-  // Image processing settings
   imageProcessingMaxConcurrency: {
     doc: 'Maximum image resizing processes to spawn concurrently. If unset (-1), set to # of CPU cores available',
     format: Number,
     env: 'imageProcessingMaxConcurrency',
     default: -1
   },
+  trackUploadArtifactDeletionEnabled: {
```

Comment:
can we call this like deleteTrackUploadArtifacts?

Reply:
done